lib/serviceruntime/roleenvironment.js

```diff
@@ -470,6 +459,17 @@ RoleEnvironment._initialize = function (callback) {
       }
     });
   };
+  
+  var getCurrentGoalState = function (finalCallback) {
+    RoleEnvironment.runtimeClient.getCurrentGoalState(function (error, goalState) {
+      if (error) {
+        callback(error);
+      } else {
+        currentGoalState = goalState;
+        getCurrentEnvironmentData(goalState, finalCallback);
```

Comment:
moved this function after it's definition of `getCurrentEnvironmentData` to comply with jshint